homeassistant/components/google_assistant/trait.py

```diff
@@ -19,6 +19,7 @@
     vacuum,
 )
 from homeassistant.components.climate import const as climate
+from homeassistant.components.manual import alarm_control_panel as alarm
```

Comment:
It's weird that we import a platform from another integration into this integration. That's normally not allowed. We should probably make the needed constant part of either the general const.py or a const.py in the alarm_control_panel integration.

@balloob?

Reply:
Should we just use the constant value directly here and then update later once the manual alarm is updated?

Reply:
If `alarm.ATTR_NEXT_STATE` is not an official part of the alarm control panel integration we should not support it in Google or other integrations. If it is, we should eb able to import the constant from the main entity.

Reply:
I am not sure if this is only related to `manual` alarm_control_panel only. The concern here is that if the manual alarm panel is used and the entity state is `pending` or `arming` Google will not know these states as they are not part of the `ArmLevels` that are synced for the entity. 
That is why I am checking for these states and use the attribute value (like `next_state`) to return one of the synced `ArmLevels`

Reply:
`ATTR_NEXT_STATE` is not a part of the official alarm control panel.

We can't have platform specific logic in the google assistant integration. We can only use the official entity states.

We should probably decide on a mapping or logic flow between the alarm control panel official states and the google assistant arm levels.